libshaderc/include/shaderc.h

```diff
@@ -24,12 +24,26 @@ extern "C" {
 #include <stdint.h>
 
 typedef enum {
-  shaderc_glsl_vertex_shader,
-  shaderc_glsl_fragment_shader,
-  shaderc_glsl_compute_shader,
-  shaderc_glsl_geometry_shader,
-  shaderc_glsl_tess_control_shader,
-  shaderc_glsl_tess_evaluation_shader
+  // Forced shader kinds. These shader kinds force the compiler to compile the
+  // source code as the specified kind of shader.
+  shaderc_glsl_forced_vertex_shader,
```

Comment:
Sorry, I think the full list should be

shaderc_glsl_vertex_shader,
shaderc_glsl_fragment_shader,
shaderc_glsl_compute_shader,
shaderc_glsl_geometry_shader,
shaderc_glsl_tess_control_shader,
shaderc_glsl_tess_evaluation_shader,
shaderc_glsl_deduce_from_pragma,
shaderc_glsl_default_vertex_shader,
shaderc_glsl_default_fragment_shader,
shaderc_glsl_default_compute_shader,
shaderc_glsl_default_geometry_shader,
shaderc_glsl_default_tess_control_shader,
shaderc_glsl_default_tess_evaluation_shader,